examples/matlab/gene_regulation/irma_model.m

```diff
@@ -52,10 +52,29 @@
          x(4)-theta{4};
          x(5)-theta{5}];
     if lifting
+        if switch_on
+            beta = SX.sym('beta', 1);
+            model.f_alg = beta - alpha(2)*(1-alpha(5));% lifted equations
+            s = [1, alpha(6);
+             1, alpha(1);
+             1, alpha(3);
+             alpha(2), beta(1);
+             1, alpha(4)];
+        else
+            beta = SX.sym('beta', 2);
+            model.f_alg = beta - [alpha(1)*(1-alpha(7)); alpha(2)*(1-alpha(5))];% lifted equations
+            s = [1, alpha(6);
+             1, beta(1);
+             1, alpha(3);
+             alpha(2), beta(2);
+             1, alpha(4)];
+        end
+        model.z_user = beta;
```

Comment:
I was thinking to name the user algebraic variables just  `z`, and internally when they get collected to call them
`z_all` 

All other variables defined by the user that enter `model` do not have subscripts (except `v_global`) and
for `f_alg`, i would prefer `g_z` and all them together `g_z_all` (as it already is)

Reply:
Rename is done.